perses/rjmc/geometry.py

```diff
@@ -455,13 +455,14 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
         if self.use_sterics:
             platform_name = 'CPU' # faster when sterics are in use
         else:
-            platform_name = 'Reference' # faster when only valence terms are in use
+            platform_name = 'CUDA' # faster when only valence terms are in use
```

Comment:
Can we avoid specifying CUDA? This means this won't work on Travis tests or on local laptops if CUDA is not available.

Is there a way to just discover the fastest platform instead?

Reply:
i guess just call `utils.get_fastest_platform` and then modify the platform if it gets a CPU with single precision.

Reply:
The problem is that returning `OpenCL` means that we don't know if it supports double precision (e.g. on `osx` or Travis). I think we actually have to instantiate a `Context` with a simple `System` (without specifying a platform) to see what the fastest platform that supports those kernels is, and then figure out what platform was assigned.

Reply:
This is fed into `configure_platform(fallback='Reference',precision='double')`

so if cuda is not there like on travis, it falls back to the reference platform incase the fastest platform doesn't support double